src/export/Model/Api/DataIntegrationClient.php

```diff
@@ -0,0 +1,361 @@
+        return $body;
+    }
+
+    /**
+     * Get the data quality summary report content from Fredhopper
+     *
+     * @param string $triggerId
+     * @param bool $isSummary
+     * @return string|null
+     */
+    public function getDataQualityReport(string $triggerId, bool $isSummary): ?string
+    {
+        $dataQualityUrl = $this->getBaseUrl(self::ENDPOINT_DQ) .
+            sprintf(($isSummary ? self::FAS_DATA_QUALITY_ENDPOINT : self::FAS_DATA_QUALITY_ZIP_ENDPOINT), $triggerId);
+        $request = $this->generateRequest($dataQualityUrl, [], Request::METHOD_GET);
+        $response = $this->sendRequest($request);
+
+        if (!$this->checkResponseStatusCode($response)) {
+            return null;
+        }
+
+        return $response['body'] ?? null;
+    }
+
+    /**
+     * Generate an HTTP request
+     *
+     * @param string $url
+     * @param array $parameters
+     * @param string $method
+     * @return Request
+     */
+    private function generateRequest(string $url, array $parameters, string $method): Request
+    {
+        $request = $this->httpClient->getRequest();
+        $request->setMethod($method);
+        $request->setUri($url);
+        if (isset($parameters['headers'])) {
+            $headers = $request->getHeaders();
+            $headers->addHeaders($parameters['headers']);
+            $request->setHeaders($headers);
+        }
+        if (isset($parameters['body'])) {
+            $request->setContent($parameters['body']);
+        }
+        if (isset($parameters['query'])) {
+            $queryParams = $request->getQuery();
+            foreach ($parameters['query'] as $name => $value) {
+                $queryParams->set($name, $value);
+            }
+            $request->setQuery($queryParams);
+        }
+        return $request;
+    }
+
+    /**
+     * Sends the given request
+     *
+     * @param Request $request
+     * @return array
+     */
+    private function sendRequest(Request $request): array
+    {
+        $auth = $this->getAuth();
+        $this->httpClient->setAuth($auth['username'], $auth['password']);
+
+        $response = $this->httpClient->send($request);
+
+        if ($this->generalConfig->getDebugLogging()) {
+            $this->logger->debug("Request response:\n $response");
+        }
+        // clear client for next request
+        $this->httpClient->reset();
+        return [
+            'status_code' => $response->getStatusCode(),
+            'headers' => $response->getHeaders()->toArray(),
+            'body' => $response->getBody()
+        ];
+    }
+
+    /**
+     * Returns whether the response status code is a 2xx status
+     *
+     * @param array $response
+     * @return bool
+     */
+    private function checkResponseStatusCode(array $response): bool
+    {
+        // Treat any response code other than 2xx as an error
+        $statusString = (string)$response['status_code'];
+        if (strlen($statusString) < 1 || $statusString[0] !== '2') {
```

Comment:
```suggestion
        if (empty($statusString) || str_starts_with($statusString, '2')) {
```

Just using empty for readability, and if this is PHP8 there's a more readable functions for strings starting with a character.